application/helpers/admin/statistics_helper.php

```diff
@@ -153,7 +153,7 @@ function createChart($iQuestionID, $iSurveyID, $type, $lbl, $gdata, $grawdata, $
 
             $counter = 0;
             foreach ($lblout as $sLabelName) {
-                $DataSet->SetSerieName(html_entity_decode($sLabelName, null, 'UTF-8'), "Serie" . $counter);
+                $DataSet->SetSerieName(html_entity_decode($sLabelName, ENT_QUOTES|ENT_SUBSTITUTE, 'UTF-8'), "Serie" . $counter);
```

Comment:
Moving from `ENT_QUOTES | ENT_SUBSTITUTE | ENT_HTML401` to `ENT_QUOTES | ENT_SUBSTITUTE` no ?

Seems it's OK, but it update something.